packages/asu-core/src/aws/providers/s3.provider.ts

```diff
@@ -0,0 +1,7 @@
+import { S3 } from "aws-sdk";
+
+export class S3Provider {
+  resolve(): S3 {
```

Comment:
do we need this stuff?

```suggestion
@injectable()
export class S3Provider {
  public resolve(): S3 {
```

Reply:
I guess not, but we should have it for consistency.